azure-kusto-data/azure/kusto/data/request.py

```diff
@@ -140,6 +167,24 @@ def application_key(self):
         """The application key to use for authentication when federated authentication is used"""
         return self._internal_dict.get(self.ValidKeywords.application_key)
 
+    @property
+    def application_certificate(self):
+        """A PEM encoded certificate private key."""

```

Comment:
Isn't it just "X509 certificate"

Reply:
https://github.com/AzureAD/azure-activedirectory-library-for-python/blob/ef3a8814540684b6fbd709a249dc7c522f94584a/adal/authentication_context.py#L244